### ISSUE TITLE
label file

### ISSUE THREAD

**Issue by zhou18** [logpai/loglizer#18]
hi i did not find rm_repeat_mlabel.txt file ,could you please where is it ?
what is the structure of this file ? is it like this?
BlockId | Label

**Comment by jimzhu** [comment]
Yes. The structure is correct. You can find it at The following link

**Comment by jimzhu** [comment]
https://github.com/logpai/loghub/tree/bba4876fb9e45c78501e598950bf5ff68dfef7bf/HDFS

**Comment by zhou18** [comment]
but i got this error:
  File "/home/z/Desktop/AdvanceTask3/part2/prjPart2/utils/data_loader.py", line 32, in hdfs_data_loader
    label_df = pd.read_csv(label_path, delimiter=r'\s+', header=None, usecols = [0], dtype =int) # usecols must be a list
  File "/home/z/.local/lib/python2.7/site-packages/pandas/io/parsers.py", line 678, in parser_f
    return _read(filepath_or_buffer, kwds)
  File "/home/z/.local/lib/python2.7/site-packages/pandas/io/parsers.py", line 446, in _read
    data = parser.read(nrows)
  File "/home/z/.local/lib/python2.7/site-packages/pandas/io/parsers.py", line 1036, in read
    ret = self._engine.read(nrows)
  File "/home/z/.local/lib/python2.7/site-packages/pandas/io/parsers.py", line 1848, in read
    data = self._reader.read(nrows)
  File "pandas/_libs/parsers.pyx", line 876, in pandas._libs.parsers.TextReader.read
  File "pandas/_libs/parsers.pyx", line 891, in pandas._libs.parsers.TextReader._read_low_memory
  File "pandas/_libs/parsers.pyx", line 968, in pandas._libs.parsers.TextReader._read_rows
  File "pandas/_libs/parsers.pyx", line 1094, in pandas._libs.parsers.TextReader._convert_column_data
  File "pandas/_libs/parsers.pyx", line 1162, in pandas._libs.parsers.TextReader._convert_tokens
ValueError: invalid literal for long() with base 10: 'blk_-1608999687919862906,Normal'

**Comment by zhou18** [comment]
when i run PCA_hdfs.py

**Comment by jimzhu** [comment]
We have updated our demo code recently for readability. Pls try the new demos. We will also release the benchmark code soon. @zhou18
